source/deploy/mobile-app-changelog.md

```diff
@@ -30,6 +30,7 @@ Mattermost mobile v2.0 is a major update to the iOS and Android apps bringing su
 Users now gain a more reliable and feature-rich application, improving their experience for collaborating with their teams on the go.
 
 ### Known Issues
+ - Landscape mode doesn't work on Android and iPhone devices.
```

Comment:
can we remove iPhones? that was a conscious decision and is exactly like that on v1.

In fact Android phones are also locked in v1, but I guess newer Android devices (the ones that fold) can be used as tablets.. it looks to me that the ones that are "tablets" are the problem